Add blog post link to dockershim removal faq (#31057)
kubernetes/website: content/en/blog/_posts/2020-12-02-dockershim-faq.md

what that means, check out the blog post
[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/).

> Also, read this [blog post](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) to check whether Dockershim deprecation affects you.

Reviewer: ```suggestion
Also, you can read [check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) to check whether it does.
```

Reviewer: The task we're linking to _isn't_ a blog post.
